dist/s_evergreen

```diff
@@ -4,6 +4,35 @@
 setup_trap
 cd_top
 
+echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
```

Comment:
I think `s_all` fails if there's anything printed to stdout/stderr by our scripts. Can we remove these echoes and only print text when an error occurs?

Reply:
yep sounds good, removed in [bba91b0](https://github.com/wiredtiger/wiredtiger/pull/10886/commits/bba91b0245ea7b608e9ae80066dd22322753022a)